config/quick-start/control-cluster/kustomization.yaml

```diff
@@ -1,8 +1,9 @@
 resources:
-  - controller
+  - ../../default
   - ../../istio
   - ../../metallb
   - ../../cert-manager
+  - ../../gateway-api
```

Comment:
I think this is fine, it does seem to make more sense here over where it was before